src/Symfony/Component/Validator/Constraints/Json.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Imad ZAIRIG <imadzairig@gmail.com>
+ */
+class Json extends Constraint
+{
+    const NOT_JSON_ERROR = '0789c8ad-2d2b-49a4-8356-e2ce63998504';
```

Comment:
Minor comment: most of the existing constraints use `INVALID_*` for these error names. Should we rename this to `INVALID_JSON_ERROR` too for consistency? Personally it looks better too.

Reply:
yes, I've just updated this, thanks :)